example_project/example_project/settings.py

```diff
@@ -130,4 +130,9 @@
 STATIC_URL = '/static/'
 STATIC_ROOT = 'staticfiles'
 
+
+TESTING = sys.argv[1:2] == ['test']
```

Comment:
i think because he doesn't check the length, a slice will always return an empty array